pkg/notify/notify.go

```diff
@@ -107,6 +113,13 @@ func (r *Receiver) Notify(data *alertmanager.Data, hashJiraLabel bool, updateSum
 			}
 		}
 
+		if r.conf.UpdateInComment {
```

Comment:
this works better for me to avoid flodding of comments if repeat interval is set low

```
		if r.conf.UpdateInComment {
			if issue.Fields.Description != issueDesc {
				retry, err := r.addComment(issue.Key, issueDesc)
				if err != nil {
					return retry, err
				}
			}
		}
```

Reply:
And add descriptionhere

```
func (r *Receiver) search(project, issueLabel string) (*jira.Issue, bool, error) {
	query := fmt.Sprintf("project=\"%s\" and labels=%q order by resolutiondate desc", project, issueLabel)
	options := &jira.SearchOptions{
		Fields:     []string{"summary", "status", "resolution", "resolutiondate", "description"},
		MaxResults: 2,
	}
```